docker/python/Dockerfile

```diff
@@ -27,6 +27,10 @@ RUN apt-get update -y && \
     git \
     ca-certificates
 
+# Install poetry
+RUN curl -sSL https://install.python-poetry.org | python3 -
+ENV PATH "/root/.local/bin:${PATH}"
```

Comment:
I'm curious: why `/root/.local/bin` vs `/usr/local/bin`?

Reply:
I had another look at the install script, and you can set $POETRY_HOME to influence the installation location. I made it `/usr/local`, so the binary will be in `/usr/local/bin/poetry`. There's a `/usr/local/venv/` that contains the actual installation, I think that's ok, but we could also install in `/usr/local/poetry` or `/opt/poetry` or something like that, and then link it into `/usr/local/bin/`.

Reply:
Actually I prefer putting it in `/usr/local/share/pypoetry` and then symlinking, updated with that.